docs/using/construction.md

```diff
@@ -0,0 +1,279 @@
+>>> model.positions
+<Quantity([[ 0.08816532 -0.00447812 -0.00147432]
+ [-0.058171   -0.03757246  0.0050985 ]
+ [-0.13500406  0.07580698  0.01761578]
+ [ 0.12650467  0.01742136  0.10122475]
+ [ 0.10164929  0.08705406 -0.06089891]
+ [ 0.1476358  -0.08945496 -0.03918502]
+ [-0.07853556 -0.09968277  0.09683283]
+ [-0.08355056 -0.10035449 -0.08158895]
+ [-0.1086939   0.1512604  -0.03762466]], 'nanometer')>
+>>> model.positions.m_as(unit.angstrom)
+array([[ 0.88165321, -0.04478118, -0.01474324],
+       [-0.58171004, -0.37572459,  0.05098497],
+       [-1.35004062,  0.75806983,  0.17615782],
+       [ 1.26504668,  0.17421359,  1.01224746],
+       [ 1.01649295,  0.87054063, -0.60898906],
+       [ 1.47635802, -0.89454965, -0.39185017],
+       [-0.78535559, -0.99682774,  0.96832828],
+       [-0.83550563, -1.00354494, -0.81588946],
+       [-1.08693898,  1.51260405, -0.3762466 ]])
+```
+
+### Box vectors
+
+Information about the periodic box of a system is stored as a unit-tagged $3×3$
+matrix, following conventional periodic box vectors and the implementation in
+[`OpenMM`](http://docs.openmm.org/latest/userguide/theory/05_other_features.html#periodic-boundary-conditions).
+
+Box vectors are represented with nanometers internally and when reported.
+However, they can be set with box vectors of other units, and can also be
+converted as desired.
+
+If box vectors are passed to the setter without tagged units, nanometers will be
+assumed. The setter will immediately tag the unitless vectors with units; the
+internal state always has units explicitly associated with with the values.
+
+If a $1×3$ matrix (array) is passed to the setter, it is assumed that these
+values correspond to the lengths of a rectangular unit cell ($a_x$, $b_y$,
+$c_z$).
+
+Box vectors are optional; if it is `None` it is implied that the `Interchange`
+object represents a non-periodic system. However, note that passing `None` to
+the `box` argument of `Interchange.from_smirnoff()` may produce a periodic
+`Interchange` if box vectors are present in the `Topology`.
+
+```pycon
+
+>>> from openff.interchange import Interchange
+>>> from openff.units import unit
+>>> import numpy as np
+>>> model = Interchange()
+>>> model.box is None
+True
+>>> model.box = np.eye(3) * 4 * unit.nanometer
+>>> model.box
+<Quantity([[4. 0. 0.]
+ [0. 4. 0.]
+ [0. 0. 4.]], 'nanometer')>
+>>> model.box = [3, 4, 5]
+<Quantity([[3. 0. 0.]
+ [0. 4. 0.]
+ [0. 0. 5.]], 'nanometer')>
+>>> model.box = [28, 28, 28] * unit.angstrom
+<Quantity([[2.8 0. 0.]
+ [0. 2.8 0.]
+ [0. 0. 2.8]], 'nanometer')>
+>>> model.box.m_as(unit.angstrom)
+array([[28.,  0.,  0.],
+       [ 0., 28.,  0.],
+       [ 0.,  0., 28.]])
+```
+
+
+[`Interchange`]: openff.interchange.Interchange
+[`Interchange.from_smirnoff()`]: openff.interchange.Interchange.from_smirnoff
+[`Interchange.topology`]: openff.interchange.Interchange.topology
+[SMIRNOFF force field]: openff.toolkit.typing.engines.smirnoff.forcefield.ForceField
```

Comment:
This bothered me too; I just tried to be consistent with the naming of the method (`from_smirnoff` when it's really `from_offtk_forcefield` or something.) I'll take another look at it; I can probably thread this needle. But reconsidering the method name might be worthwhile, if its not too late; it feels like its been named to describe the parameter handlers used under the hood rather than how it is actually used by consumers of the API. If in the future, for example, `ForceField` were extended to also support other kinds of force field, or another project pops up that implements SMIRNOFF, this method might find itself awkwardly named.